src/ui/server/controllers/insurance/business/company-details/validation/company-details/index.ts

```diff
@@ -10,9 +10,11 @@ import { RequestBody } from '../../../../../../../types';
 const validation = (responseBody: RequestBody, errors: object) => {
   let updatedErrors = errors;
 
-  for (let i = 0; i < companyDetailsResponseRules.length; i += 1) {
-    updatedErrors = companyDetailsResponseRules[i](responseBody, updatedErrors);
-  }
+  /* eslint-disable no-unused-vars, prettier/prettier */
+  companyDetailsResponseRules.forEach((rule: (ruleData: object, ruleErrors: ValidationErrors) => ValidationErrors) => {
+    /* eslint-enable no-unused-vars, prettier/prettier */
```

Comment:
Please remove `inline` es-lint rule.

Reply:
@abhi-markan We have to keep this in unfortunately, same reason as above https://github.com/UK-Export-Finance/exip/pull/199#discussion_r1055611819

But the good news is we now only have this inline eslint rule in 3 places rather than having it littered in every single validation function.